src/applications/personalization/profile/components/direct-deposit/DirectDepositV2.jsx

```diff
@@ -92,21 +110,23 @@ const DirectDeposit = ({
         <ReactCSSTransitionGroup
           transitionName="form-expanding-group-inner"
           transitionAppear
-          transitionAppearTimeout={500}
-          transitionEnterTimeout={500}
-          transitionLeaveTimeout={500}
+          transitionAppearTimeout={bankInfoUpdatedAlertSettings.FADE_SPEED}
+          transitionEnterTimeout={bankInfoUpdatedAlertSettings.FADE_SPEED}
+          transitionLeaveTimeout={bankInfoUpdatedAlertSettings.FADE_SPEED}
         >
           {!!recentlySavedBankInfo && (
-            <AlertBox
-              status={ALERT_TYPE.SUCCESS}
-              backgroundOnly
-              className="vads-u-margin-top--0 vads-u-margin-bottom--2"
-              scrollOnShow
-            >
-              We’ve updated your bank account information for your{' '}
-              <strong>{recentlySavedBankInfo}</strong> and your next payment
-              will go to your new account.
-            </AlertBox>
+            <div data-testid="bankInfoUpdateSuccessAlert">
+              <AlertBox
```

Comment:
I'm sure you thought the same thing, but it would have been nice if we could have just added the `data-testid` to `AlertBox`, but I guess that would be an unsupported prop